lib/broken_record/schedulers/external_job_scheduler.rb

```diff
@@ -15,6 +15,7 @@ def run
 
         # Don't run in parallel, just utilize the callback functionality of the parallel gem
         Parallel.each(jobs, { finish: finish_callback, in_processes: 0 }) do |job|
+          ActiveRecord::Base.connection.reconnect!
```

Comment:
Would `verify!` work here as well? Looks like `reconnect!` forces a new connection while `verify!` checks before reconnecting.

Reply:
`verify!` also works here but because Parallel guarantee disconnects us from our database for each job run, I didn't think we would need to check if the database was up.